policy_set.go

```diff
@@ -61,7 +64,9 @@ type PolicySet struct {
 	Name           string    `jsonapi:"attr,name"`
 	Description    string    `jsonapi:"attr,description"`
 	Global         bool      `jsonapi:"attr,global"`
+	PoliciesPath   string    `jsonapi:"attr,policies-path,omitempty"`
 	PolicyCount    int       `jsonapi:"attr,policy-count"`
+	VCSRepo        *VCSRepo  `jsonapi:"attr,vcs-repo,omitempty"`
```

Comment:
The ordering satisfies my ODC now, but the tags still contain `omitempty` 😉

Reply:
I thought this was copied from `WorkSpace` but I guess not, will update!